src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  --grey: rgba(225, 231, 237, 1);
+  --text: rgb(61, 78, 97);
+}
+
+body {
+  margin-top: 0;
+}
+
+.input {
+  width: 100%;
+  outline: none;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid var(--grey);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  color: var(--text);
+  font-family: Avenir-Heavy, sans-serif;
+  font-weight: 900;
+  box-sizing: border-box;
+}
+
+.input:not(:last-child) {
+  margin-bottom: 20px;
+}
+
+.input::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 400;
+}
+
+[data-qa*="big"] {
```

Comment:
I appreciate you that you know how to use `data-qa` for styling. 
Consider use class names for this task for more practice with reusable classes